### PR TITLE
Fix the usage of the SelfValidating passport

### PR DESCRIPTION
**Pull request by javiereguiluz**
I tested #112 changes and it didn't work ... the changes in this PR made my app work again, but I'm not sure it it's correct as a general solution. Thanks!

### CODE REVIEW

File: src/Security/ConnectAuthenticator.php [hunk 1]
```diff
@@ -140,7 +140,9 @@ public function authenticate(Request $request): PassportInterface
             throw new AuthenticationServiceException('The user provider must return a UserInterface object.');
         }
 
-        $passport = new SelfValidatingPassport(new UserBadge($localUser), []);
+        $passport = new SelfValidatingPassport(new UserBadge($localUser->getUuid(), function (string $symfonyConnectUuid) {
```
Comment (fabpot) [inline src/Security/ConnectAuthenticator.php:143]: Isn't it the default behavior?

Comment (javiereguiluz) [reply src/Security/ConnectAuthenticator.php]: I'm not entirely sure. But, at least my own Symfony app, only works if I make this change.

Maybe @wouterj or @chalasr can clarify if I'm doing something wrong. Thanks!

Comment (wouterj) [reply src/Security/ConnectAuthenticator.php]: Yes, it is. Have you configured the correct user provider, @javiereguiluz ?

Comment (javiereguiluz) [reply src/Security/ConnectAuthenticator.php]: I've made further changes to remove this code, which indeed is the default behavior. Thanks!

**Review by wouterj — APPROVED** [review]

**Comment by fabpot** [comment]
And tagged

**Pull request merged**